sretoolbox/utils/threaded.py

```diff
@@ -16,10 +16,9 @@
 Threading abstractions.
 """
 
-import functools
-from multiprocessing.dummy import Pool as ThreadPool
+from concurrent.futures import ThreadPoolExecutor
 
-from sretoolbox.utils.exception import SystemExitWrapper
+from sretoolbox.utils.concurrent import pmap
 
 
 def run(func, iterable, thread_pool_size, return_exceptions=False, **kwargs):
```

Comment:
You added such a nice doc string to pmap. Could you update this one as well?

Reply:
updated